ansible_collections/arista/avd/roles/eos_l3ls_evpn/templates/fabric/mlag/leaf-mlag-spanning-tree.j2

```diff
@@ -1,4 +1,8 @@
 {# Leaf mlag disable spanning tree #}
 {% if leaf.mlag == true %}
+{%     if type == 'l3leaf' %}
   no_spanning_tree_vlan: 4093-4094
+{%     elif type == 'l2leaf' %}
```

Comment:
just do `else` here, so we avoid any future issues. This only runs for mlag devices, so it should be safe.

Reply:
Thanks, updated!